test/e2e/e2e.go

```diff
@@ -306,6 +341,29 @@ func CreatePod(clients *test.Clients, pod *corev1.Pod, _ logging.FormatLogger, c
 	return nil
 }
 
+// SendFakeEventToChannel will create fake CloudEvent and send it to the given channel
+func SendFakeEventToChannel(clients *test.Clients, senderName string, body string, eventType string, encoding string, channel *v1alpha1.Channel, ns string, logf logging.FormatLogger, cleaner *test.Cleaner) error {
```

Comment:
Rather than taking in `senderName, body, eventType, encoding`, pass in a `test.CloudEvent`.

Reply:
Done.